src/Error.ts

```diff
@@ -28,7 +28,7 @@ export class BrsError extends Error {
             formattedLocation = `${location.file}(${location.start.line},${location.start.column},${location.end.line},${location.end.line})`;
         }
 
-        return `${formattedLocation}: ${this.message}`;
+        return `${formattedLocation}: ${this.message}\n`;
```

Comment:
Perhaps I'm missing something, but I'm not sure what this has to do with suppressing the "something terrible happened" errors.  What's the motivation behind this change?

Reply:
I'll check again, but I've found that errors were somehow all printed on a single line.

Reply:
Also, we print that string of non-fatal errors☝️ at the beginning of every unit test run. Is there some way we could let the user know that those errors are non-fatal? Maybe we could add `Warning:` before them?

I've heard feedback from users that it's difficult to tell which error is the one that's causing the failure when those non-fatal errors are printed directly above the fatal one.

Reply:
I appreciate that it's wrong to add the newline when formatting the error. I couldn't narrow why the errors are printed without newlines...

Reply:
> Also, we print that string of non-fatal errors☝️ at the beginning of every unit test run. Is there some way we could let the user know that those errors are non-fatal? Maybe we could add `Warning:` before them?

Filed #467 to track that!

> I've found that errors were somehow all printed on a single line.

Agreed, I've seen that too!  Looking deeper into it, we probably should move that into `getLoggerUsing` so that the error messages themselves don't need to worry about how they're logged.  If we switch back to `console.error`, we'd wind up with extra newlines if we append `\n` here!